aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -29,6 +29,9 @@
 
 namespace at {
 namespace native {
+// PI constant for bfloat16 according to https://en.wikipedia.org/wiki/Bfloat16_floating-point_format#Special_values
+template<>
+constexpr c10::BFloat16 pi<c10::BFloat16> = c10::BFloat16(0x4049, c10::BFloat16::from_bits());
```

Comment:
Why is this not in native/Math.h ?

Reply:
It should be moved to the header file, but not sure whether `native/Math.h` is the best place, since it does not include BFloat16.h by default, does it?